### PR TITLE
Eia comparisons

### PR DESCRIPTION
**Pull request by eringold**
Pull request overview
---------------------

Adds plotting of monthly energy compared to EIA monthly electricity and natural gas

 - Fixes #ISSUENUMBERHERE (IF RELEVANT)

### Pull Request Author

This pull request makes changes to (select all the apply):
 - [ ] Documentation
 - [ ] Infrastructure (includes singularity image, buildstock batch, dependencies, continuous integration tests)
 - [ ] Sampling
 - [ ] Workflow Measures
 - [ ] Upgrade Measures
 - [ ] Reporting Measures
 - [x] Postprocessing

Author pull request checklist:

 - [x] Tagged the pull request with the appropriate label (documentation, infrastructure, sampling, workflow measure, upgrade measure, reporting measure, postprocessing) to help categorize changes in the release notes.
 - [x] 10k+ test run
 - [ ] Change documentation written
 - [ ] Measure documentation written
 - [ ] ComStock documentation updated
 - [ ] All new and existing tests pass the CI

### Review Checklist

This will not be exhaustively relevant to every PR.
 - [x] Perform a code review on GitHub
 - [x] All related changes have been implemented: data and method additions, changes, tests
 - [x] If fixing a defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] Reviewed change documentation
 - [x] Ensured code files contain License reference
 - [ ] Results differences are reasonable
 - [ ] CI status: all tests pass

#### ComStock Licensing Language - Add to Beginning of Each Code File
```
# ComStock™, Copyright (c) 2023 Alliance for Sustainable Energy, LLC. All rights reserved.
# See top level LICENSE.txt file for license terms.
```

### CODE REVIEW

**Comment by eringold** [comment]
Sample of generated plots (126 in total) using baseline_data_10k run:

com_eia_electricity_consumption

com_eia_electricity_consumption_by_division

com_eia_monthly_electricity_consumption_by_division_for_mountain
com_eia_monthly_electricity_consumption_by_in state_for_ca
com_eia_monthly_electricity_consumption_by_in state_for_id
com_eia_natural_gas_consumption
com_eia_monthly_natural_gas_consumption_by_division_for_mountain
com_eia_monthly_natural_gas_consumption_by_in state_for_az
com_eia_monthly_natural_gas_consumption_by_in state_for_ak
com_eia_monthly_natural_gas_consumption_by_in state_for_me

**Comment by mdahlhausen** [comment]
@eringold I'm getting an error running the test: 
```

up_res = up_res.with_columns(pl.col(col).str.to_lowercase().replace({"false": False, "true": True}, default=None))
E                   AttributeError: 'Expr' object has no attribute 'replace'
```

File: postprocessing/comstockpostproc/comstock.py
Comment (mdahlhausen) [inline postprocessing/comstockpostproc/comstock.py]: csvTODOANDREW is not a valid filetype

**Review by mdahlhausen — DISMISSED** [review]
Address minor

**Comment by mdahlhausen** [comment]
@eringold good to merge after resolving the merge conflict with the main branch

**Review by mdahlhausen — APPROVED** [review]

**Pull request merged**
